tooling/template-base/components/CourseLayout.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { usePageContext } from '../renderer/usePageContext';
+import CourseSidebar from './CourseSidebar';
+
+export default function CourseLayout({
+  title,
+  coursePages
+}: {
+  title?: string;
+  coursePages: Array<JSX.Element>;
+}) {
+  const pageContext = usePageContext();
+  let pageQuery = 1;
+
+  const { search } = pageContext.urlParsed;
+  if (search?.page) {
+    const pageInt = parseInt(search.page, 10);
+    pageQuery = isNaN(pageInt) ? 1 : pageInt;
```

Comment:
this lets negative / zero slip through. we may want to just go ahead and check for `coursePages[pageInt]` for ultimate validity?

Reply:
Felt a little messy to do `coursePages[pageInt - 1]` cause of index differences, so opted for checking if `pageInt < 1` instead if that works.